azkaban-web-server/src/main/java/azkaban/webapp/servlet/ScheduleServlet.java

```diff
@@ -671,6 +676,11 @@ private void ajaxScheduleFlow(HttpServletRequest req,
     }
 
     List<SlaOption> slaOptions = null;
+    try {
+      slaOptions = getSlaOptionList(req, flowName);
+    } catch (Exception e) {
```

Comment:
Hmm, generally speaking, if user requests wrongly, server should return 4xx(for example, 400 Bad Request).
If there are some bugs, server should return 5xx.

But before you fix it in that way, I prefer to add stacktrace(for example, https://github.com/azkaban/azkaban/pull/890, https://github.com/azkaban/azkaban/pull/905)

Regarding stacktrace, I will to send pull request later.